README.md

```diff
@@ -1,9 +1,55 @@
 # JupyterHub Administration to Empower Research Communities
 
-This lesson teaches the fundamentals of JupyterHub administration and user support.
+This lesson is aimed at Hub Champions and teaches the fundamentals of hub administration and user support for 2i2c managed JupyterHubs in the cloud. Delivered over two synchronous workshops, two hours in length a week apart. The time in between is dedicated to self-guided study with asynchronous support (via email 2i2c support / online Slack? To be discussed).
 
-## Note 
-This lesson is under active development and not ready for teaching yet.
+> [!WARNING]
+> This lesson is under active development and not ready for teaching yet.  
+
+> [!INFO]
+> TODO: Link to the Spanish repo.
+
+## Lesson objectives
+
+After attending this training, Hub Champions will be able to:
+
+- explain how interactive cloud computing can empower your user community
+- control user authentication and access to your hub
+- monitor usage and cloud billing costs
+- apply software environments through containerization
+- navigate the filesystem and transfer data to and from the hub
+- provide user support and troubleshoot common issues
+
+## Episode objectives
+
+1. :star: Recognising the benefits your hub can provide for your user community
+   - identify use cases of your user community
+   - distinguish which hub features can support these use cases
+   - understand your role and capacity as a Hub Champion to administer and support your hub
+1. Administering your hub
+   - add users to the hub with GitHub
+   - configure server options (CPU, GPU, RAM)
+   - stop and restart a server
+   - monitor usage and estimate cloud billing costs
+1. Applying software environments
+   - explain the value of reproducibility with containerized software environments
+   - select a pre-configured image container for your hub
+   - pull a custom container to your hub
```

Comment:
The pre-configured environments are likely Jupyter and RStudio. I can think of nice exercise by having learners launch an already configured and available 'Linux Desktop Environment' to demonstrate the versatility available here.

Reply:
Perhaps "pull a custom container to your hub" could be interpreted broadly. My intention here was to narrow the focus and walk through hub champions how to use the "Other" profile option to copy and paste in a publicly available image tag from https://quay.io/organization/2i2c